package.json

```diff
@@ -5,22 +5,24 @@
   "description": "Monorepo root package for lit.dev",
   "author": "Google LLC",
   "license": "BSD-3-Clause",
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "bootstrap": "lerna bootstrap --ci",
     "build": "wireit",
     "start": "run-p -r start:main start:playground start:fake-github",
-    "start:main": "LITDEV_ENV=local MODE=main lerna run start --scope=lit-dev-server --stream",
-    "start:playground": "LITDEV_ENV=local MODE=playground lerna run start --scope=lit-dev-server --stream",
+    "start:main": "LITDEV_ENV=local MODE=main npm run start -w lit-dev-server",
+    "start:playground": "LITDEV_ENV=local MODE=playground npm start -w lit-dev-server",
     "start:fake-github": "wireit",
-    "dev": "cd packages/lit-dev-server && npm run build && cd ../lit-dev-content && (npm run dev:build:assets watch & npm run dev:build:eleventy:watch & node ../lit-dev-tools-esm/lib/dev-server.js --open)",
+    "dev": "cd lit-dev-server && npm run build && cd ../lit-dev-content && (npm run dev:build:assets --watch & npm run dev:build:eleventy:watch & node ../lit-dev-tools-esm/lib/dev-server.js --open)",
     "format": "prettier . --write",
     "format:check": "prettier . --check",
-    "nuke": "rm -rf node_modules && npm ci && lerna exec 'rm -rf node_modules' && lerna bootstrap --ci",
-    "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap",
-    "test:unit": "lerna run test --scope lit-dev-tools-cjs --scope lit-dev-tools-esm --stream",
-    "test:integration": "lerna run test:integration --stream",
-    "test:integration:update-golden-screenshots": "lerna run test:integration:update-golden-screenshots --stream",
-    "test:links": "lerna run test:links --stream"
+    "nuke": "rm -rf node_modules packages/*/node_modules && npm ci",
```

Comment:
Is this command necessary anymore? I thought `npm ci` was supposed to put node_modules in exactly the state described by package-lock.json

Reply:
Done in https://github.com/lit/lit.dev/pull/863